### PR TITLE
Polish cli commands

### PR DESCRIPTION
**Pull request by nlsdfnbch**

### CODE REVIEW

**Comment by LefterisJP** [comment]
For a moment there I thought you implemented Polish translations of the CLI commands ... what with the P being uppercase and all.

**Comment by nlsdfnbch** [comment]
Sprawdź to, @ulope

File: scenario_player/main.py [hunk 1]
```diff
@@ -43,11 +43,14 @@
...
-    file_name = f"scenario-player-{sub_command}_{datetime.now():%Y-%m-%dT%H:%M:%S}.log"
     directory = data_path
     if scenario_fpath:
-        directory = directory.joinpath(scenario_fpath.stem)
-    return str(directory.joinpath(file_name))
+        file_name = f"scenario-player-{sub_command}_{scenario_fpath.stem}_{datetime.now():%Y-%m-%dT%H:%M:%S}.log"
+        directory = directory.joinpath('scenarios', scenario_fpath.stem)
+    else:
+        file_name = f"scenario-player-{sub_command}_{datetime.now():%Y-%m-%dT%H:%M:%S}.log"
+    dir = str(directory.joinpath(file_name))
```
Comment (ulope) [inline scenario_player/main.py]: Lint police: `dir` is a builtin

File: scenario_player/main.py [hunk 2]
```diff
@@ -292,21 +303,35 @@ def pack_n_latest_logs_for_scenario_in_dir(scenario_name, scenario_log_dir: Path
...
+    return message
+
+
+def post_to_rocket_chat(fpath, **rc_payload_fields):
     try:
         user = os.environ['RC_USER']
         pw = os.environ['RC_PW']
         room_id = os.environ['RC_ROOM_ID']
     except KeyError as e:
         raise RuntimeError('Missing Rocket Char Env variables!') from e
```
Comment (czepluch) [inline scenario_player/main.py]: Should be `Rocket Chat`

**Pull request merged**